src/Subclasses/User.php

```diff
@@ -22,6 +22,14 @@ class User extends ParseUser implements Authenticatable
 {
     use CastsParseProperties;
 
+	/**
+     * @return string
+     */
+    public function getAuthIdentifierName()
+    {
+        return $this->getUsername();
```

Comment:
Yeah no worries I realised I'd made a mistake with return 'username' as well when I reread the docs